controller/iteration_blackbox_test.go

```diff
@@ -707,7 +719,7 @@ func createSpaceAndRootAreaAndIterations(t *testing.T, db application.DB) (space
 		errCreateOwner := app.Identities().Create(context.Background(), owner)
 		require.Nil(t, errCreateOwner)
 		spaceObj = space.Space{
-			Name:    testsupport.CreateRandomValidTestName("CreateSpaceAndRootAreaAndIterations-"),
+			Name:    testsupport.CreateRandomValidTestName("foo-"),
```

Comment:
This was needed to not have the inserted UUID shortened. Otherwise the `compareWithGoldenUUIDAgnostic` wouldn't pick it up as a UUID ;)